test/js/animation.js

```diff
@@ -318,12 +318,12 @@ suite('animation', function() {
     tick(1000);
     var a = document.body.animate([], 2000);
     a.finish();
-    assert.equal(a.startTime, 0);
+    assert.equal(a.startTime, -1000);
```

Comment:
We called a.finish() which seeks currentTime to 2000 so startTime jumps to -1000 given that timeline.currentTime is 1000.